File: 351002/vorobei/DataAccess/Models/Story.cs
namespace DataAccess.Models
{
    public class Story : BaseEntity
    {
        public int CreatorId { get; set; }
        public string Title { get; set; } = string.Empty;
        public string Content { get; set; } = string.Empty;
        public DateTime Created {  get; set; }
        public DateTime Modified { get; set; }
    }
}

File: 351002/Bubich/t310/app/models/story.py
from dataclasses import dataclass
from datetime import datetime
from typing import Optional
from app.models.base import BaseEntity

@dataclass
class Story(BaseEntity):
    writer_id: Optional[int] = None
    title: str = ""
    content: str = ""
    created: Optional[datetime] = None
    modified: Optional[datetime] = None